src/sentry/static/sentry/app/components/events/interfaces/richHttpContent.jsx

```diff
@@ -32,7 +32,13 @@ const RichHttpContent = React.createClass({
             : [[k, val]] // key has single value
         );
       }, [])
-      .sort(function([keyA], [keyB]) {
+      .sort(function([keyA, valA], [keyB, valB]) {
+        // if keys are identical, sort on value
+        if (keyA === keyB) {
+          keyA = valA;
+          keyB = valB;
```

Comment:
sorry for even commenting this but i'd rather 
 `return valA < valB ? -1 : 1; `
here. doesn't matter.